docs/appendices/release-notes/5.6.1.rst

```diff
@@ -76,3 +76,6 @@ Fixes
 
 - Fixed an issue that caused filtering by ``primary keys`` under ``NOT``
   predicate to return invalid results.
+
+- Fixed an issue that caused a ``where`` clause equivalent to ``null`` to
+  return invalid results.
```

Comment:
Could you make this a bit more verbose? Based on just this I'd have no idea what was actually fixed or what was broken

Reply:
Thank you, updated